File: test/qr.js
// This is free software for the public good of a permacomputer hosted at
// permacomputer.com, an always-on computer by the people, for the people.
// One which is durable, easy to repair, & distributed like tap water
// for machine learning intelligence.
//
// The permacomputer is community-owned infrastructure optimized around
// four values:
//
//   TRUTH      First principles, math & science, open source code freely distributed
//   FREEDOM    Voluntary partnerships, freedom from tyranny & corporate control
//   HARMONY    Minimal waste, self-renewing systems with diverse thriving connections
//   LOVE       Be yourself without hurting others, cooperation through natural law
//
// This software contributes to that vision by enabling code execution across 42+ programming languages through a unified interface, accessible to all.
// Code is seeds to sprout on any abandoned technology.

const QRCode = require('qrcode');
QRCode.toString('unsandbox-qr-ok', { type: 'utf8', margin: 0 }, function(err, str) {
    const rows = str.trim().split('\n').length;
    console.log('QR:unsandbox-qr-ok:ROWS:' + rows);
});

File: test/qr.go
package main

import (
	"fmt"
	qrcode "github.com/skip2/go-qrcode"
)

func main() {
	qr, err := qrcode.New("unsandbox-qr-ok", qrcode.Medium)
	if err != nil {
		panic(err)
	}
	rows := len(qr.Bitmap())
	fmt.Printf("QR:unsandbox-qr-ok:ROWS:%d\n", rows)
}
